Alpaca.Markets/Enums/TimeInForce.cs

```diff
@@ -4,13 +4,14 @@
 namespace Alpaca.Markets;
 
 /// <summary>
-/// Supported order durations in Alpaca REST API.
+/// Supported order durations in Alpaca REST API. See https://alpaca.markets/docs/trading/orders/#time-in-force for
```

Comment:
Please, use HTML-like hyperlinks with the `href` attribute and user-friendly text - it works better for both VS IntelliSense and for auto-generated documentation.